Resources/Prototypes/Reagents/Materials/materials.yml

```
@@ -123,4 +123,12 @@
   unit: materials-unit-piece
   icon: { sprite: Objects/Misc/reagent_fillings.rsi, state: powderpile }
   color: "#A9A9A9"
-  price: 0
\ No newline at end of file
+  price: 0
+
+- type: material
+  id: Diamond
+  name: materials-diamond
+  unit: materials-unit-piece
+  icon: { sprite: Objects/Materials/materials.rsi, state: diamond }
+  color: "#80ffff"
+  price: 2.5 # This should probably be a lot more but I'll leave that to a different pr
```

```suggestion
  price: 10 
```
live dangerously, lad

---

live dangerously